misc/launchd/org.nixos.nix-daemon.plist.in

```diff
@@ -17,7 +17,7 @@
     <array>
       <string>/bin/sh</string>
       <string>-c</string>
-      <string>/bin/wait4path @bindir@/nix-daemon &amp;&amp; /nix/var/nix/profiles/default/bin/nix-daemon</string>
+      <string>/bin/wait4path /nix/store/ &amp;&amp; /nix/var/nix/profiles/default/bin/nix-daemon</string>
```

Comment:
it's probably best to wait on the exact path that is going to be used to avoid race conditions:
```suggestion
      <string>/bin/wait4path /nix/var/nix/profiles/default/bin/nix-daemon &amp;&amp; /nix/var/nix/profiles/default/bin/nix-daemon</string>
```

Reply:
Sure, updating and re-pushing. One sec.